### ISSUE TITLE
Wrong setting in config

### ISSUE THREAD

**Issue by psykhon** [synthetos/g2#134]
#define A_RADIUS (M1_TRAVEL_PER_REV/(2*3.14159628))
should be 
#define A_RADIUS (M4_TRAVEL_PER_REV/(2*3.14159628))
ans such

**Comment by justinclift** [comment]
Taking a look through some older issues to close out ones that don't seem useful any more.

With this, I've just taken a look at the [settings_default.h](https://github.com/synthetos/g2/blob/cb7bf178e76ea1ce855dd940bfbc37eb8df2b2d4/g2core/settings/settings_default.h#L548) file, and it's still M1.  So this issue may still be relevant.

Thinking about it... the A axis does seem to be commonly used with motor 4.  So having the A axis calculations default to using motor 4 seems to make sense.  I can create a simple PR to update that if it's useful. :smile:

**Comment by aldenhart** [comment]
Makes sense to make Motor4 the default.

Please keep in mind that some machines (like Othermill) use different motor/axis mappings for physical wiring reasons. I only bring that up here so it's brought to people's attention. We may want a comment that the setting is only a default. But perhaps that obvious by the filename :-)

**Comment by justinclift** [comment]
Yeah, I'll create the PR for the settings_default.h file, but leave the others alone under the assumption they've already been tweaked for their intended purpose.

Which reminds me, I should look into the details of those other machines, then add them to the new "[Who uses g2core?](https://github.com/synthetos/g2/wiki/g2core-in-use)" page.
